### PR TITLE
DSP-1245: Add more example metadata

### PR DESCRIPTION
**Pull request by BalduinLandolt**
DSP-1245

### CODE REVIEW

File: example/example-metadata-maximal.ttl [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+    <hasDateCreated> "2021-01-04"^^xsd:date ;
+    <hasDateModified> "2021-01-05"^^xsd:date ;
+    <hasDatePublished> "2021-01-06"^^xsd:date ;
+    <hasDistribution> [ a schema:DataDownload ;
+            schema:url "download.dsp.dasch.swiss/9997" ] ;
+    <hasDocumentation> [ a schema:URL ;
+            schema:propertyID [ a schema:PropertyValue ;
+                    schema:propertyID "github.com" ] ;
+            schema:url "http://www.github.com/maximum/media/readme.md" ],
+        "totally not documented"^^xsd:string ;
```
Comment (mpro7) [inline example/example-metadata-maximal.ttl]: Is this the example of string and URL type in the same property?

**Review by mpro7 — APPROVED** [review]
Thanks!
Apart of **Some checks were not successful**, looks fine.

**Review by mpro7 — CHANGES_REQUESTED** [review]
Actually changing my mind ;)
all props are missing now `repository` part, so I'm getting `"http://ns.dasch.swiss/hasAbstract"` instead of `"http://ns.dasch.swiss/repository#hasAbstract"`

**Review by mpro7 — COMMENTED** [review]
Ok, IRI part was fixed, thanks!

**Comment by mpro7** [comment]
This is how `Place` object looks like in old example:
```
{
    "@type": "https://schema.org/Place",
    "https://schema.org/url": {
        "@type": "https://schema.org/URL",
        "https://schema.org/propertyID": {
            "@type": "https://schema.org/PropertyValue",
            "https://schema.org/propertyID": "Geonames"
        },
        "https://schema.org/url": "https://www.geonames.org/2017370/russian-federation.html"
    }
}
```
and this is how it looks in minimal version:
```
{
    "@type": "https://schema.org/Place",
    "https://schema.org/url": {
        "@type": "https://schema.org/URL",
        "https://schema.org/propertyID": {
            "@type": "https://schema.org/PropertyValue",
            "https://schema.org/propertyID": "Geonames",
            "https://schema.org/url": "https://www.geonames.org/2661604"
        }
    }
}
```
The bottom url property is now inside `propertyID`. Which one is correct?

**Comment by BalduinLandolt** [comment]
You're right, the old one is correct.

I'll fix it in the tool. Thanks for spotting!

**Comment by mpro7** [comment]
@BalduinLandolt could you please add multiple address, it's `0-n` for `Person` and I didn't find it in both of examples.

EDIT:
Not a case anymore. We agreed to make both properties equal = `0-1`

**Comment by mpro7** [comment]
> @BalduinLandolt could you please add multiple address, it's `0-n` for `Person` and I didn't find it in both of examples.
> 
> EDIT:
> Not a case anymore. We agreed to make both properties equal = `0-1`

But maybe you can within this PR align above cardinalities.
Otherwise other things seems to be ok.

**Review by mpro7 — APPROVED** [review]

**Pull request merged**